### ISSUE TITLE
[feature request] can the command attribute of El dropdown item component support symbol type

### ISSUE THREAD

**Issue by ZhouJiang** [element-plus/element-plus#2005]
### Existing Component
Yes

### Component Name
el-dropdown-item

### Description
el-dropdown-item  组件 command 属性可否支持  Symbol 类型

**Comment by JeremyWuuuuu** [comment]
Yes, we can. I wonder why do you need to use Symbol in this case?
Could you provide some insights about your use cases?
